modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -5,252 +5,639 @@ endif::[]
-| Keytab a| String |  Path to keytab file associated with the `principal`. |  | x
-| Service Name a| String |  The Kerberos principal name that the Kafka broker runs as. |  | x
-| Additional JAAS Properties a| Object |  Additional properties such as a key-value pair that you need to set in `sasl.jaas.config` and that you usually include in the JAAS configuration file. |  | 
-|===
-
-[[kafka-consumer-config_ssl-kafka-consumer-connection]]
-===== Kafka SSL Consumer Connection
 
 ====== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Consumer Partitions a| Number |  The number of partitions to use for the consumer. |  1 | 
-| Group Id a| String |  A unique string that identifies the consumer group this consumer belongs to. |  | x
-| Bootstrap Servers a| String |  Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. The same as the `bootstrap.servers` value you must provide to Kafka clients (producer and consumer). |  | x
-| Additional Properties a| Object |  Additional properties such as a key-value pair that you need for your connection. |  | 
+| Bootstrap server URLs a| Array of String |  The list of servers to bootstrap the connection with the Kafka cluster. This can be a partial list of the available servers. |  | x
+| Batch size a| Number |  The producer attempts to batch records together into fewer requests whenever multiple records are being sent to the same partition. This helps performance on both the client and the server. This configuration controls the default batch size in bytes. No attempt will be made to batch records larger than this size. Requests sent to brokers will contain multiple batches, one for each partition with data available to be sent. A small batch size makes batching less common and may reduce throughput (a batch size of zero disables batching entirely). A very large batch size may use memory more wastefully as a buffer of the specified batch size is always allocated in anticipation of additional records. |  `16` | 
+| The batch size unit of measure. a| Enumeration, one of:
+
+** BYTE
+** KB
+** MB
+** GB |  The unit of measure for the batch size scalar. |  `KB` | 
+| Buffer size a| Number |  The total bytes of memory the producer can use to buffer records waiting to be sent to the server. If records are sent faster than they can be delivered to the server the producer will block for `max.block.ms` after which it throws an exception. This setting should generally correspond to the total memory the producer will use, but is not hard-bound since not all memory the producer uses is used for buffering. Some additional memory is used for compression (if compression is enabled) as well as for maintaining in-flight requests. The default value in the Kafka documentation is 33554432 (32MB), but this should be capped to align with expected values for Mule instances in cloudhub (v0.1 core) |  1 | 
```

Comment:
Add a comma after "server." Maybe change "hard-bound" to "exact"? I would also change "since" to "because" here and elsewhere, but this is a nice-to-have. Should "cloudhub" be initial-capped?